### PR TITLE
 format CoordinateTransformer along with QgsQuick

### PR DESCRIPTION
**Pull request by 3nids**
also add transform context

### CODE REVIEW

**Comment by 3nids** [comment]
this needs `QgsCoordinateTransformContext::context` which was added in 3.4.
@m-kuhn what do we do?

File: src/qml/PositionInformationView.qml [hunk 1]
```diff
@@ -13,6 +13,7 @@ Item {
     id: _ct
     sourceCrs: CrsFactory.fromEpsgId(4326)
     sourcePosition: positionSource.position.coordinate
+    transformContext: qgisProject.transformContext
```
Comment (3nids) [inline src/qml/PositionInformationView.qml]: does this seem right?

File: src/qml/TransformedPositionSource.qml [hunk 2]
```diff
@@ -12,6 +12,7 @@ PositionSource {
     id: _ct
     sourceCrs: CrsFactory.fromEpsgId(4326)
     sourcePosition: _pos.coordinate
+    transformContext: qgisProject.transformContext
```
Comment (3nids) [inline src/qml/TransformedPositionSource.qml]: same here

**Comment by m-kuhn** [comment]
> this needs QgsCoordinateTransformContext::context which was added in 3.4.

3.4 or 3.2?
3.2 should be ready in https://github.com/opengisch/QField/pull/295

**Comment by 3nids** [comment]
3.4 :(
https://qgis.org/api/classQgsCoordinateTransform.html#a07014bf094a7515f4f3cfbbcff112e93

**Comment by m-kuhn** [comment]
I'll create a new sdk image soon then

**Comment by m-kuhn** [comment]
> I'll create a new sdk image soon then

It's no longer trivial, the expression function help grew too big and the crosscompiler complains. I have to postpone a bit.

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://opengis.ch/download/qfield/ci-builds/qfield-dev-310-2d478a73c7ee88d02a78638ac2b87be7d565c651-armv7.apk) and [x86](https://opengis.ch/download/qfield/ci-builds/qfield-dev-310-2d478a73c7ee88d02a78638ac2b87be7d565c651-x86.apk)

**Pull request merged**